odata-csdl/12 Action and Function.md

```diff
@@ -181,22 +181,49 @@ Absence of the attribute means `false`.
 
 ## ##subsec Entity Set Path
 
-Bound actions and functions that return an entity or a collection of
-entities MAY specify an entity set path if the entity set of the
-returned entities depends on the entity set of the binding parameter
+Actions and functions bound to an entity or a collection of
+entities that return an entity or a collection of
```

Comment:
If $k>0$, the binding parameter need not be an entity (collection), it can also be a complex instance.

Example: The return type is the entity type `Country`, the binding parameter type is the complex type `Address` with a containment navigation property `Country`. The entity set path is `bindingParameter/Country`.

When the operation is invoked on `Customers(2)/Address`, the returned entities MUST belong to the implicit entity set `Customers(2)/Address/Country`.